### DISCUSSION TITLE
Can you add support for pygame to VS 2022

### DISCUSSION THREAD

**Discussion by vsfeedback** [Microsoft/PTVS#7567] [Enchancements]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/可以在视觉工作室中加入pygame/10154037)._

---
可以在Visual Studio 2022中加入pygame吗？....................................................................................

---
#### Feedback Bot on 9/22/2022, 01:03 AM: 

(private comment, text removed)

**Comment by cwebster-99** [comment]
What do you mean add support for pygame? Pygame can be installed in VS currently
